launch/auton_sim.launch

```diff
@@ -3,18 +3,32 @@
  -->
 <launch>
     <arg name="rvizconfig" default="$(find mrover)/config/rviz/auton_sim.rviz" />
+    <arg name="world_name" default="world.world" />
+
     <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
 
     <!-- launch rover core nodes -->
     <include file="$(find mrover)/launch/rover_core.launch" />
 
     <!-- launch the simulator -->
-    <include file="$(find mrover)/launch/simulator.launch" />
+    <include file="$(find mrover)/launch/simulator.launch">
+        <arg name="world_name" value="$(arg world_name)" />
+    </include>
 
     <!-- launch auton -->
     <include file="$(find mrover)/launch/auton.launch" >
         <arg name="sim" value="true" />
         <arg name="use_ekf" value="true" />
         <arg name="ekf_start_delay" value="10.0" />
     </include>
+
+    <node name="stereo_odometry" pkg="rtabmap_ros" type="stereo_odometry" output="screen" launch-prefix="bash -c 'sleep 10; $0 $@'">
```

Comment:
we should probably put this node in auton.launch with an arg to turn it on/off, since we also want to be able to run this IRL